src/test/java/com/prgrms/nabmart/domain/cart/CartTest.java

```diff
@@ -0,0 +1,52 @@
+package com.prgrms.nabmart.domain.cart;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.prgrms.nabmart.domain.user.User;
+import com.prgrms.nabmart.domain.user.exception.NotExistUserException;
+import com.prgrms.nabmart.global.fixture.UserFixture;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class CartTest {
+
+    User givenUser;
+    Cart givenCart;
+
+    @BeforeEach
+    void setUp() {
+        givenUser = UserFixture.user();
+    }
+
+    @Nested
+    @DisplayName("장바구니 생성 시")
+    class NewCartTest {
+
+        @Test
+        @DisplayName("성공")
+        void newCart() {
+            // given
+
+            // when
+            givenCart = new Cart(givenUser);
+
+            // then
+            assertThat(givenCart.getUser()).isEqualTo(givenUser);
+        }
+
+        @Test
+        @DisplayName("예외 : 사용자가 null")
+        void throwExceptionWhenUserIsNull() {
+            // given
+
+            // when
+
+            // then
+            assertThatThrownBy(() -> new Cart(null))
+                .isInstanceOf(NotExistUserException.class);
```

Comment:
```suggestion
            // given
            User nullUser = null;
            
            // when
            Exception exception = catchException(() -> new Cart(nullUser));
                
            // then
            exception.isInstanceOf(NotExistUserException.class);
```

이렇게 바꾸면 테스트의 given when then 절이 명확해질거 같습니다!

Reply:
오 꿀팁 감사합니다!